pkg/restic/exec_commands.go

```diff
@@ -20,17 +20,22 @@ import (
 	"bytes"
 	"encoding/json"
 	"fmt"
+	"sync"
```

Comment:
looks like this isn't used, so need to remove

Reply:
good catch, think it was left over from a rebase conflict